lib/commands/project/project.js

```diff
@@ -39,8 +39,15 @@ var FRAMEWORKS = {
   sails:   { source: 'sails' }
 };
 
-var ASSERTIONTYPES = ['expect', 'should', 'assert'];
-var TESTMODULES = ['supertest', 'request'];
+var TEST_ASSERTION_TYPES = ['expect', 'should', 'assert'];
+var TEST_MODULES = ['supertest', 'request'];
+var TEST_DEPENDENCIES = {
+  'z-schema': {'z-schema': '^3.12.0'},
```

Comment:
if i do the ```{name: version}``` hash, how do i check if the property already exists in the package.json? i can't check against the ```name``` if its the actual key in the object to be added?